### ISSUE TITLE
Accessibility issue in Terra-tabs

### ISSUE THREAD

**Issue by SushmithaBN** [cerner/terra-framework#841]
# Issue Description
In the latest version of terra-tabs, we can see few accessibility violations when it is inspected [here](https://engineering.cerner.com/terra-framework/raw/tests/terra-tabs/tabs/tabs/controlled-tabs) 
The same violations are seen while running WDIO test for accessibility

## Additional Context / Screenshots
![image](https://user-images.githubusercontent.com/42537313/63049395-357b0e00-bef6-11e9-8ab7-d66b8191d4eb.png)

**Comment by bjankord** [comment]
Seems like the blue text color doesn't match contrast.
The axe tool is generating a false positive. Axe is comparing color to background-image colors which background-image is used for the blue bar. The actually tab background is #fff.

**Comment by nramamurth** [comment]
cc / @neilpfeiffer
